sdk/iotoperations/Azure.ResourceManager.IoTOperations/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Need to add some contents in the following sections. Besides, if there is a target release date, don't forget to update it.
See [here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges) for details.